src/Kunstmaan/SeoBundle/Resources/config/services.yml

```diff
@@ -36,3 +36,9 @@ services:
         arguments: ["@doctrine.orm.entity_manager", "@kunstmaan_admin.clone.helper"]
         tags:
             - { name: kernel.event_listener, event: kunstmaan_admin.postDeepCloneAndSave, method: postDeepCloneAndSave }
+
+    kunstmaanseobundle.seo_management_menu_adaptor:
+        class: Kunstmaan\SeoBundle\Helper\Menu\SeoManagementMenuAdaptor
+        arguments: ["@service_container", "@security.context"]
```

Comment:
don't think you use the security context? the contructor only accepts a container